features/api/command/run.feature

```diff
@@ -60,13 +60,13 @@ Feature: Run command
     Then the specs should all pass
 
   Scenario: Non-existing executable
-    Given a file named "bin/cli" does not exist
+    Given a file named "bin/cli-app" does not exist
```

Comment:
I'm already confused. I'd call it "bin/nonexisting" or something more suggestive like that. Is that ok?